icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -5341,6 +5341,17 @@ void MeasureFormatTest::TestBadArg() {
 
 void MeasureFormatTest::TestEquality() {
     UErrorCode status = U_ZERO_ERROR;
+
+    // First verify that the equality operators of class Measure work.
```

Comment:
I like my test a little better.

Reply:
What I find problematic with your test is that it compiles (and runs) just fine, without even the smallest little warning, also with `-std=c++20`, so it does in no way help verifying that the fix actually fixes what it's supposed to.

Reply:
I exercise both operator== and operator!=, just like your test, but with more combinations.
How does your test do better? Is it because you have a `Measure` on the right hand side?

Reply:
> Is it because you have a `Measure` on the right hand side?

Yes, that's the important part.

Reply:
Ok. Ideas:
- You could rebase your PR and add your test as well.
- You or I could improve my test code so that it uses both UObject and Measure on the right hand side.

Reply:
Done.